servatrice/scripts/linux/maint_replays

```diff
@@ -1,8 +1,9 @@
 #!/bin/bash
 
-# SCHEDULE WITH CRONTAB BASED ON TIME PERIOD REPLAYS SHOULD BE SAVED UNTIL (EX: SCHEDULE ONCE A WEEK TO KEEP A WEEKS WORTH OF REPLAYS IN THE DB)
-
+# SCHEDULE WITH CRONTAB DAILY
 DBNAME="servatrice"					#set this to the database name used
 TABLEPREFIX="cockatrice"			#set this to the prefix used for the table names in the database (do not inclue the _)
 SQLCONFFILE="./mysql.cnf" 			#set this to the path that contains the mysql.cnf file
-mysql --defaults-file=$SQLCONFFILE -h localhost -e 'truncate table ""$DBNAME"".""$TABLEPREFIX""_replays;truncate table ""$DBNAME"".""$TABLEPREFIX""_replays_access'
+mysql --defaults-file=$SQLCONFFILE -h localhost -e 'delete from servatrice.cockatrice_games where time_finished < DATE_SUB(now(), INTERVAL 8 DAY)'
+mysql --defaults-file=$SQLCONFFILE -h localhost -e 'delete from servatrice.cockatrice_replays where id_game NOT IN (select id from servatrice.cockatrice_games)'
+mysql --defaults-file=$SQLCONFFILE -h localhost -e 'delete from servatrice.cockatrice_replays_access where id_game NOT IN (select id from servatrice.cockatrice_games)'
```

Comment:
I can see your reasoning.  The question I guess I would have is "Should the replays and access table have entries in it that the games table does not have?"

I was approaching the problem with the mind set that if the games table does not contain the game information in it, the access and replay tables should be pruned / cleaned to match.  That being said the important query in the script is the first delete and the other two could technically be executed at any time.  It just so happens that they are executed consecutively.

Either solution is better than the original truncating of the tables IMO.